pkgs/applications/science/electronics/horizon-eda/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, pkgconfig, sqlite, libyamlcpp, libuuid, gnome3, epoxy, librsvg, zeromq, cppzmq, glm, libgit2, curl, boost, python3, opencascade, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+	name = "horizon-eda-${version}";
+	version = "20190227";
+
+	src = fetchFromGitHub {
+		owner = "carrotIndustries";
+		repo = "horizon";
+		rev = "810fc8d1f59096ba3e1aac9d4b2187b1010b202a";
+		sha256 = "0qxw3vsbf9cg6b3pp3z1k2fjl8xrnjdr2gcncnr5jzd1mix6yd4y";
+	};
+
+	# Version in specific format, created by: "git log -1 --pretty="format:%h %ci %s"
```

Comment:
Updating this by hand seems like it'd get annoying after a while. What about just taking part of `src.rev` or does it have to have that format? Or at least use git within the build to produce the thing.

Also note that overriding `patchPhase` prevents users from passing additional patches via `patches`; not necessarily a problem, but something to be aware of.